src/config.js

```diff
@@ -8,7 +8,9 @@ const configs = [
     SIDECHAIN: {
       NAME: "Leap Testnet",
       RPC: "https://testnet-node.leapdao.org",
-      DAI_ADDRESS: "0xD2D0F8a6ADfF16C2098101087f9548465EC96C98",
+      DAI_ADDRESS: "0x1f89Fb2199220a350287B162B9D0A330A2D2eFAD",
+      DAI_SYMBOL: "₲",
+      DAI_SHORTNAME: "GÖLR",
```

Comment:
should we add those values also for the `toco2` domain in the config?

Reply:
That's a good question :)
Can we move this one out to XSS (or maybe even unpaid) issue ?

Reply:
There are some more values like "BRIDGE_ADDRESS" and "UNISWAP" addresses, which is incorrect, though unused in PlanetA.

Reply:
I've updated other values though, but would be cool if they are working correctly on that domain.